static/docs/use-cases/how-to-store-files-in-dvc.md

```diff
@@ -0,0 +1,129 @@
+	    new file:   .dvc/config
+    $ git commit -m "Initialize dvc"
+```
+
+Start tracking assets with dvc:
+
+```sh
+    $ dvc add assets
+```
+
+DVC will move your data to `.dvc/cache`, create links to your workspace, add
+your data to `.gitignore` and will also create a special `.dvc` file with
+information about the data:
+
+```sh
+    $ tree .dvc/cache
+    .dvc/cache
+    ├── 18
+    │   └── d51187e9602848514499047bb6de7f
+    ├── 25
+    │   └── 79801f17b00b0014d4e2a4847e3064.dir
+    ├── 3c
+    │   └── af879afa11fc750cac4e15721a8741
+    ├── 82
+    │   └── 986e112ecefabbcfbaf960e2c8fb36
+    └── f8
+        └── 3c01a5634a9e7b8a04177a5533f67b
+
+    5 directories, 5 files
+    $ cat assets.dvc
+    md5: 5fdf01d92fd021acb587e78eb04743f7
+    outs:
+    - cache: true
+      md5: 2579801f17b00b0014d4e2a4847e3064.dir
+      path: assets
+```
+
+NOTE: For more info on dvc file format, see [`DVC File Format`](https://dvc.org/doc/user-guide/dvc-file-format).
+
+Commit your changes:
+
+```sh
+    $ git status
+    ...
+    Untracked files:
+        .gitignore
+        assets.dvc
+    $ git add .gitignore assets.dvc
+    $ git commit -m "Track assets with dvc"
+```
+
+To share your data with others you need to setup dvc remote repository.
+DVC supports a variety of remote types (see
+[`dvc remote`](https://dvc.org/doc/commands-reference/remote) for more info),
+but for this example let's use AWS S3 remote to store your cache. To add
+your S3 repository and set it as a default one run:
+
+```sh
+    $ dvc remote add -d myremote s3://mybucket/mycache
+    Setting 'myremote' as a default remote.
+```
+
+This will add `myremote` to your `.dvc/config`. Commit your changes and push
+your code:
+
+```sh
+    $ git add .dvc/config
+    $ git push
+```
+
+Now, to upload your cache use:
+
+```sh
+    $ dvc push
+    (1/5): [##############################] 100% 3caf879afa11fc750cac4e15721a8741
+    (2/5): [##############################] 100% f83c01a5634a9e7b8a04177a5533f67b
+    (3/5): [##############################] 100% 2579801f17b00b0014d4e2a4847e3064.dir
+    (4/5): [##############################] 100% 82986e112ecefabbcfbaf960e2c8fb36
+    (5/5): [##############################] 100% 18d51187e9602848514499047bb6de7f
+```
+
+In order for your collagues to obtain the cache, they will simply need to run:
+
+```sh
+   $ git pull
+   $ dvc pull
+   (1/5): [##############################] 100% 3caf879afa11fc750cac4e15721a8741
+   (2/5): [##############################] 100% f83c01a5634a9e7b8a04177a5533f67b
+   (3/5): [##############################] 100% 2579801f17b00b0014d4e2a4847e3064.dir
+   (4/5): [##############################] 100% 82986e112ecefabbcfbaf960e2c8fb36
+   (5/5): [##############################] 100% 18d51187e9602848514499047bb6de7f
+```
+
+This will download the cache files that are currently used in the branch and
+setup all the needed links from the cache to workspace, creating `assets`
+directory and its contents.
```

Comment:
This is a great use case. But I have a feeling it is a bit advanced. Why don't we separate this use case into two:
1. how to store data files
2. how DVC works

The first one should not have any details about `.dvc` dir or might have just a single reference to a magic `.dvc` directory which contains actual data and DVC configuration.

Reply:
Ok, I'll throw out the details for this PR and will come back to more in-depth docs later.